app/views/roles/_current_role_holder_with_biography.html.erb

```diff
@@ -3,10 +3,13 @@
     <%= render "govuk_publishing_components/components/heading", {
       text: t("roles.headings.current_holder"),
       lang: t_fallback("roles.headings.current_holder"),
+      margin_bottom: 2,
     } %>
 
     <%= render "govuk_publishing_components/components/heading", {
       text: role.current_holder["title"],
+      heading_level: 3,
+      font_size: "m",
```

Comment:
what does font-size 'm' mean?

Reply:
This is an option for the component, which causes it to display the heading at a medium font size. You can see this documented here: https://components.publishing.service.gov.uk/component-guide/heading/with_margin

Reply:
That is fine